lms/djangoapps/mobile_api/testutils.py

```diff
@@ -92,9 +92,9 @@ def reverse_url(self, reverse_args=None, **kwargs):
             reverse_args.update({'username': kwargs.get('username', self.user.username)})
         return reverse(self.REVERSE_INFO['name'], kwargs=reverse_args)
 
-    def url_method(self, url, **kwargs):  # pylint: disable=unused-argument
+    def url_method(self, url, qargs={}, **kwargs):  # pylint: disable=unused-argument
```

Comment:
Use default of `None`.  This creates one empty dictionary that will be shared by all callers of `url_method` that don't specify a qargs value.  If it gets modified anywhere along the way, all callers will see that modification.